tests/unit/test_zos_console.py

```diff
@@ -19,4 +20,11 @@ def setUp(self):
     def test_object_should_be_instance_of_class(self):
         """Created object should be instance of Console class."""
         console = Console(self.session_details)
-        self.assertIsInstance(console, Console)
\ No newline at end of file
+        self.assertIsInstance(console, Console)
+
+    @mock.patch('requests.Session.send')
+    def test_get_response_should_return_messages(self, mock_send_request):
+        """Getting z/OS Console response messages on sending a response key"""
+        mock_send_request.return_value = mock.Mock(headers={"Content-type": "application/json"}, status_codes=200)
```

Comment:
Please change `status_codes` -> `status_code` to fix the failing test 🙂

Reply:
done